schedule/yast/nvme.yaml

```diff
@@ -26,6 +26,7 @@ schedule:
   - installation/reboot_after_installation
   - installation/grub_test
   - installation/first_boot
+  - console/sle15_workarounds
```

Comment:
Don't we want to add other modules here too, like `system_prepare` etc.? Because the plan is to get rid of this module.

Reply:
I didn't want to extend more the execution time, only that one is needed. When that will happen I guess we would have to check all the test coverage and make more homogeneous (but I was not aware of that plan to create a ticket for that for example).

Reply:
Nope, idea is to reuse existing modules like the one, I've mentioned above. Sure, we can merge this PR, but last time execution time was exactly the point not to schedule modules like the ones which trigger btrfs maintenance.
So ok, let's merge this workaround and handle this in that ticket.